public/app/features/dashboard-scene/pages/DashboardScenePage.tsx

```diff
@@ -22,6 +22,7 @@ export interface Props
 export function DashboardScenePage({ route, queryParams, location }: Props) {
   const params = useParams();
   const { type, slug, uid } = params;
+  const starparam = params["*"] // used to select a dashboard by path
```

Comment:
This breaks viewing of existing dashboards, I think we'd do smth like `path: '/dashboard/repo/:slug/:path',` in the routes and then use `path` from `prams` here.

Reply:
right, but `:path` does not allow slashes, and as a query parameter it is kinda gross

Reply:
yea, I think can change that route to `'/dashboard/repo/:slug/:path*' so it can capture the full path